docs/website/docs/guides/deployment-configurations/gpu-rocm.md

```diff
@@ -30,8 +30,6 @@ or [Linux](https://rocm.docs.amd.com/en/latest/deploy/linux/quick_start.html).
 
 #### :octicons-package-16: Download the compiler from a release
 
-!!! note "Currently ROCm is **NOT supported** for the Python interface."
```

Comment:
Include steps here, so the section isn't empty. See the other guides for text you can copy/paste:

https://github.com/iree-org/iree/blob/916bb8847ee3fff722c0fece0ae8dba291c73fed/docs/website/docs/guides/deployment-configurations/gpu-vulkan.md?plain=1#L79-L114

If the text is the same in each guide, we could use a snippet maybe:
* https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/?h=snippet#snippets
* https://facelessuser.github.io/pymdown-extensions/extensions/snippets/

Reply:
There seems to be some information missing to use the runtime, so need to figure out what else to document before we can land this.

Reply:
Thanks for pointing me to using a snippet for this. I sent a bigger patch which supersedes this one and that makes use of a snippet.